requirements/base.txt

```diff
@@ -1,4 +1,4 @@
-Django==3.2.16
+Django~=3.2.17
```

Comment:
probably makes more sense to use `==`.

Reply:
If we use [Compatible release](https://peps.python.org/pep-0440/#compatible-release), we should use `~=`, 
If we use [Version matching](https://peps.python.org/pep-0440/#version-matching), we should use `==`
Here we give full version `3.2.17`, so I think we need to use `Version matching` as `Django==3.2.17`. 
If we use `Compatible release`, should use `Django~=3.2`

Reply:
already updated earlier using `==`.